src/main/java/fi/otavanopisto/kuntaapi/server/integrations/casem/CaseMClient.java

```diff
@@ -0,0 +1,96 @@
+package fi.otavanopisto.kuntaapi.server.integrations.casem;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+
+import org.apache.http.client.utils.URIBuilder;
+
+import fi.otavanopisto.kuntaapi.server.integrations.GenericHttpCache;
+import fi.otavanopisto.kuntaapi.server.integrations.GenericHttpClient;
+import fi.otavanopisto.kuntaapi.server.integrations.GenericHttpClient.Response;
+import fi.otavanopisto.casem.client.ApiResponse;
+
+/**
+ * API Client for CaseM
+ * 
+ * @author Antti Leppä
+ */
+@Dependent
+public class CaseMClient implements fi.otavanopisto.casem.client.ApiClient {
+
+  private static final String INVALID_URI_SYNTAX = "Invalid uri syntax";
+
+  @Inject
+  private Logger logger;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the "logger" field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1450)